docs/rules/no-duplicate-imports.md

```diff
@@ -12,59 +14,51 @@ import { find } from 'module';
 
 ## Rule Details
 
+This rules requires that all imports from a single module exists in a single `import` statement.
+
 This inspection reports any duplicated module in an import statement.
```

Comment:
I can make the change, but it really just repeats the above sentence, being more explicit about what it does